pallets/weights/src/pallet_protocol_fee.rs

```diff
@@ -52,10 +52,14 @@ use polymesh_runtime_common::{RocksDbWeight as DbWeight, Weight};
 /// Weights for pallet_protocol_fee using the Substrate node and recommended hardware.
 pub struct WeightInfo;
 impl pallet_protocol_fee::WeightInfo for WeightInfo {
-    fn change_coefficient() -> Weight {
-        (17_369_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
+    // Storage: ProtocolFee Coefficient (r:0 w:1)
+    fn change_coefficient(_n: u32, _d: u32, ) -> Weight {
+        (27_515_000 as Weight)
+            .saturating_add(DbWeight::get().writes(1 as Weight))
     }
-    fn change_base_fee() -> Weight {
-        (20_133_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
+    // Storage: ProtocolFee BaseFees (r:0 w:1)
+    fn change_base_fee(_b: u32, ) -> Weight {
```

Comment:
```suggestion
    fn change_base_fee() -> Weight {
```
Same here, the benchmark doesn't need to be parameterized.